bundle/manifests/node-healthcheck-operator.clusterserviceversion.yaml

```diff
@@ -223,15 +223,29 @@ spec:
     Remediation” API to instantiate the configured remediation template which
     specifies the mechanism/controller to be used.
 
-    ### Failure handling: Self Node Remediation
-    By default NHC depends on the “Self Node Remediation” (SNR) operator, which
-    is installed automatically.
+    ### Failure handling: External Remediators
+    The Medik8s team developed few remediators for handling node failure:
```

Comment:
in order to prevent "Medik8s" and yet another sed command, what about:
"We offer..."
or even more common:
"There are... "

Also, IMHO "multiple" would sound a bit more positive than "few"

->  "There are multiple remediators..."

WDYT?